db/migrate/20180123140634_remove_limit_ldap_filter.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveLimitLdapFilter < ActiveRecord::Migration[5.1]
+  def up
+    change_column 'auth_sources', :ldap_filter, :text, :limit => nil
+  end
+
+  def down
+    change_column 'auth_sources', :ldap_filter, :string, :limit => 255
```

Comment:
The previous schema allowed `:null => true` as well

Reply:
@tbrisker I think that's by default, `t.string "ldap_filter", limit: 255` - if you see schema.rb only `:null => false` is ever specified